src/redux/books.js

```diff
@@ -0,0 +1,29 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { v4 as uuid4 } from 'uuid4';
+
+const initialState = [];
+
+const bookSlice = createSlice({
+  name: 'books',
```

Comment:
- [ ] Hi @bhskr44, you're doing great in this project :+1: but I think we need to follow the project's requirements. We need to define [action types for adding(ADD_BOOK) and removing(REMOVE_BOOK) a book](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Define%20action%20types%20for%20adding%20and%20removing%20a%20book.) and the value of each action type needs to follow the [duck pattern](https://github.com/erikras/ducks-modular-redux) :+1: 
- [ ] [Next](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=array%20of%20books.-,Export%20Action%20Creators%20for%20your%20actions.,-Write%20your%20reducer)